ReactWindows/ReactNative.Shared/Views/Text/TextDecorationLine.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.Serialization;
+
+namespace ReactNative.Views.Text
+{
+    /// <summary>
+    /// TextDecorationLine values.
+    /// </summary>
+    public enum TextDecorationLine
+    {
+        /// <summary>
+        /// Text has no line decoration.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Text is Underlined.
+        /// </summary>
+        Underline,
+        /// <summary>
+        /// Text is Stroke out.
+        /// </summary>
+        LineThrough,
+        /// <summary>
+        /// Text is both Underlined and Stroke out.
+        /// </summary>
+        [EnumMember(Value = "underline line-through")]
+        UnderlineLineThrough
```

Comment:
When making decision on changing EnumHelpers I was looking onto "textAlign". Logically it is same as "textDecorationLine" and they both should be treated in same way. But "textDecorationLine" contains single value with space in it, which makes it impossible to use existing approach. So, either it must be an exceptional enum, treated differently (How many such enums can we possibly have?) or the approach should be changed. I've chosen the latter. 
I am not against "switch" so if you insist, that's ok. But, please note, Reflection does not have "that" bad impact on performance if it happens once. Yes, if we have 100 000 of enums, then we will see the difference, but currently we have ~20 total and maybe it will grow up to 100 or 200 at most.